doc/api/worker_threads.md

```diff
@@ -453,6 +454,7 @@ added: v10.5.0
 
 An integer identifier for the referenced thread. Inside the worker thread,
 it is available as [`require('worker_threads').threadId`][].
+This value is unique for each `Worker` instance inside a single process.
```

Comment:
"`Worker` instance" sounds like what I get back from `new Worker`.

So, for my own understanding, if I do

```js
const w1 = new Worker(somepath);
const w2 = new Worker(somepath);

w1.threadId === w2.threadId // this might be `true`?
```

If that's correct, is there a way to get a truly unique id?

Reply:
@SimenB This sentence is basically supposed to say that `w1.threadId === w2.threadId` is never true, i.e. `new Worker()` always creates a new thread id.

Do you have ideas for better phrasing?

Reply:
Right. No, I think this is fine as is, reading it again that's exactly what you say. I tumbled down a rabbit hole and somehow thought a single `new Worker()` might manage multiple threads (mixing it up with Jest's worker pooling).

So I think your sentence here is perfect, thanks!